main.cpp

```diff
@@ -51,22 +51,322 @@ If you need inspiration for what to write, take a look at previously approved st
 /*
  UDT 1:
  */
+ #include <iostream>
+#include<string>
+#include<cstdlib>
+#include <random>
+#include <iostream>
+using namespace std;
+
```

Comment:
Do not use `using` directives at the global scope.
constrain them to function scope only
- [ ] revise